Formula/minia.rb

```diff
@@ -19,8 +19,11 @@ class Minia < Formula
 
   def install
     mkdir "build" do
-      args = std_cmake_args
-      args << "-DSKIP_DOC=1"
+      args = std_cmake_args + %W[
+        -DSKIP_DOC=1
+        -DCMAKE_C_COMPILER=/usr/bin/#{OS.mac? ? "clang" : "gcc"}
+        -DCMAKE_CXX_COMPILER=/usr/bin/#{OS.mac? ? "clang++" : "g++"}
+      ]
```

Comment:
These change defeats superenv, and on Linux uses the host-provided compiler rather than the compiler provided by Homebrew, which is not desirable.